tests/v1/user_deactivation_test.py

```diff
@@ -97,7 +78,7 @@ def error_user_deactivation(test_db):
         "confirmation": False
     }, headers={'Authorization': f'Bearer {access_token}'})
     assert confirmation_false.status_code == 400
-    assert confirmation_false.json()['detail'] == 'Confirmation required to deactivate account'
+    assert confirmation_false.json()['message'] == 'Confirmation required to deactivate account'
 
 
     unauthorized = client.patch('/api/v1/users/accounts/deactivate', json={
```

Comment:
this route is not correct: /api/v1/users/accounts/deactivate

Reply:
The route is correct. It works fine
Or what do you mean please??